updater.py

```diff
@@ -34,33 +36,84 @@ def buildFromSource(tag):
     # Move binary
     os.system("mv gitea "+settings.gtfile)
 
+def is_tool(name):
```

Comment:
On the current master there is a new file for the functions. 
If you would rebase your branch, you could get the up-to-date version and put your functions into the functions file.

Reply:
done